docs/source/deployguide/use_CA.md

```diff
@@ -0,0 +1,284 @@
+
+For reference, here is a sample [peer configuration file](https://github.com/hyperledger/fabric/blob/release-2.1/sampleconfig/core.yaml). And here is a sample [ordering node configuration file](https://github.com/hyperledger/fabric/blob/release-2.1/sampleconfig/orderer.yaml).
+
+Note that these configuration files ask for the location of the relevant local MSP folder. For the peer, this is defined through the `mspConfigPath`. For the orderer, it is the `LocalMSPDir`. The folders found in this location will be used to define the local MSP of the node, including the private key the node will use when signing its actions as well as the public key of at least one admin of the node.
+
+The TLS certificates, on the other hand, are defined individually, rather than pointing to a folder, and can be found in the `TLS settings` section of the YAML. This means that TLS certificates do not need to be kept in a strict folder structure like the local MSP (relevant in particular to users who will be using an external CA to generate TLS certificates --- use the sample YAML files as a guide to what these certificates are used for). When you enroll a node with the TLS CA, the generated TLS public key can be found in the `/keystore` folder and the TLS private key can be found in the `/signcerts` folder. When you stand up a node that is enabled for TLS, you need to point to these files from the the relevant fields in the YAML config file.
+
+As with all of the configuration parameters in the YAML file of a node, you have the option to specify the `msp` folder and TLS certificate locations either in the YAML itself or through the use of environment variables.
+
+Note: **it is a best practice to mount these folders (volumes) external to the container where the node itself is running. This will allow the certificates to be used to create a new node should the node container go down, become corrupted, or is restarted.**
+
+For a look at a sample local MSP, check out [MSP structure](https://hyperledger-fabric.readthedocs.io/en/master/membership/membership.html#msp-structure). Note that you will not receive all of these certificates back simply by enrolling a peer identity. You will need, for example, to create the `users` subfolder and put the public certificate of the identity that will be administering the node in the folder prior to bootstrapping. You will also need an operations certificate (depending on the configuration of your network, this might come from a separate operations CA). For more information about the operations service, check out [The Operations Service](https://hyperledger-fabric.readthedocs.io/en/release-2.1/operations_service.html).
+
+Here is a sample local MSP as it might look the the node has been enrolled and the additional fields have been added:
```

Comment:
`as it might look the the node has been enrolled` -> `as it might look when the node has been enrolled`